src/vmm/src/config/mod.rs

```diff
@@ -60,16 +71,24 @@ impl fmt::Display for ConversionError {
 }
 
 /// Guest memory configurations.
-#[derive(Clone, Debug, Default, PartialEq)]
+#[derive(Clone, Debug, PartialEq)]
 pub struct MemoryConfig {
     /// Guest memory size in MiB.
     pub size_mib: u32,
 }
 
-impl TryFrom<String> for MemoryConfig {
+impl Default for MemoryConfig {
+    fn default() -> Self {
+        MemoryConfig {
+            size_mib: 256 as u32,
```

Comment:
nit: you can use `256u32` instead so that you do not need the conversion.

Reply:
sure.